client/src/core/analytics.js

```diff
@@ -21,13 +21,7 @@ const dimensions = {
 };
 
 let client_id;
-const get_client_id = () => {
-  if (_.isUndefined(client_id)) {
-    throw "Error: Google Analytics has not been initialized";
-  } else {
-    return client_id;
-  }
-};
```

Comment:
Oh, maybe, leave this as a warning `console.warn(...)`, just to help with debugging if something in analytics breaks in the future.